src/FEValues/face_integrals.jl

```diff
@@ -1,9 +1,9 @@
 """
-    reference_face_to_face(point::Vec{N, T}, cell_T::Type{<:AbstractRefShape}, face::Int) 
+    element_face_transformation(point::Vec{N, T}, cell_T::Type{<:AbstractRefShape}, face::Int) 
 
 Transform point from face's reference (N-1)D coordinates to ND coordinates on the cell's face.
 """
-reference_face_to_face
+element_face_transformation
```

Comment:
Sorry, I got this backwards. Reading the docstring I think it should be rather `face_to_element_transformation`.

Reply:
Nah It's my bad I missed it :D
Will rename once back to branch